.evergreen/config.in.yml

```diff
@@ -819,7 +819,9 @@ functions:
           rm -f ./prepare_client_encryption.sh
 
           VERSION=${VERSION} DRIVERS_TOOLS=${DRIVERS_TOOLS} \
-            bash ${PROJECT_DIRECTORY}/.evergreen/prepare-crypt-shared-lib.sh
+            . ${PROJECT_DIRECTORY}/.evergreen/prepare-crypt-shared-lib.sh
+
+          echo "setting CRYPT_SHARED_LIB_PATH: $CRYPT_SHARED_LIB_PATH"
 
           MONGODB_URI="${MONGODB_URI}" CRYPT_SHARED_LIB_PATH="${CRYPT_SHARED_LIB_PATH}" \
```

Comment:
It is passed explicitly to the next script.

Reply:
@durran I think this is because evergreen interprets `${CRYPT_SHARED_LIB_PATH}` as an evergreen expansion here, before bash even sees it. Using `$CRYPT_SHARED_LIB_PATH` without curly braces might work?

Reply:
Actually, even just dropping the variable from this line here should be a good fix, since `prepare-crypt-shared-lib.sh` exports it into the environment already